datumaro/plugins/data_formats/voc/base.py

```diff
@@ -45,11 +45,11 @@
 T = TypeVar("T")
 
 
-class _VocBase(SubsetBase):
+class VocBase(SubsetBase):
     def __init__(
         self,
         path: str,
-        task: VocTask,
+        task: Optional[VocTask] = VocTask.voc,
```

Comment:
```suggestion
        task: VocTask = VocTask.voc,
```

Is `None` for task is something intentional? If not, we should have these lines on line 61 along with the suggestions.

```python
        if not isinstance(task, VocTask):
            raise DatumaroImportError(f"The task must be an instance of {VocTask} but {task} is given.")
```

Reply:
Thank you for the suggestion. I think `task` should be optional for now. We plan to turn this out as mandatory in the near future.